python/tvm/micro/contrib/zephyr.py

```diff
@@ -703,6 +709,21 @@ def write(self, data, timeout_sec):
             raise TransportClosedError()
         return self.fd_transport.write(data, timeout_sec)
 
+    def _qemu_check_stdout(self):
+        for line in self.proc.stdout:
+            line = str(line)
+            _LOG.debug(line)
+            if "[QEMU] CPU" in line:
+                self._queue.put(self._qemu_ready_msg)
```

Comment:
i think we need to push an indicator of success (QEMU startup message found) or failure (didn't find that message; probably `make` error'd out). rather than push Qready, you could push e.g. True in this case and False if the loop exits (e.g. in a finally block with an "if i didn't push anything yet" check).